src/app/examples/examples.state.ts

```diff
@@ -0,0 +1,24 @@
+import { createFeatureSelector, ActionReducerMap } from '@ngrx/store';
+
+import { todosReducer } from './todos/todos.reducer';
+import { TodosState } from './todos/todos.model';
+import {
+  stockMarketReducer,
+  StockMarketState
+} from './stock-market/stock-market.reducer';
+
+export const FEATURE_NAME = 'examples';
+
+export const reducers: ActionReducerMap<ExamplesState> = {
+  todos: todosReducer,
+  stockMarket: stockMarketReducer
+};
+
+export const selectExamples = createFeatureSelector<any, ExamplesState>(
```

Comment:
Also here, we're in lazy loaded feature module and we're defining its state interface  and feature selector which accepts `AppState` and `FeatureState`. I am not completely sure what would happen if I imported `AppState` interface across the lazy boundary, would it be that just the interface gets included but bundles stay the same without some crazy duplication? I guess yes but haven't tried it yet...

Also the other way around, how can we get hold of this `ExamplesState` interface in `app.model.ts` without pulling in the rest of lazy loaded module content.

Ideas / experience with this @timdeschryver ?